exporter/collector/internal/logsutil/logsutil.go

```diff
@@ -0,0 +1,20 @@
+// Copyright 2023 OpenTelemetry Authors
```

Comment:
is this the right header for this repo?

Reply:
good catch, I actually copied this from another file in our repo so we may want to do an audit. looks like this is in a lot of places